python/__init__.py.in

```diff
@@ -95,3 +95,20 @@ def add_deprecation_warnings():
 
 
 add_deprecation_warnings()
+
+
+################################################################################
+# add support for IPython
+
+
+try:
+    from IPython import get_ipython
+
+    def f3d_image_repr_png(f3d_img: Image):
+        return f3d_img.save_buffer()
```

Comment:
would it be safer to specify the format instead of relying on the default value, or at lest more explicit?